tensorflow-core/tensorflow-core-api/src/gen/annotations/org/tensorflow/op/RiscOps.java

```diff
@@ -0,0 +1,798 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ==============================================================================
+//
+// This class has been generated, DO NOT EDIT!
+//
+package org.tensorflow.op;
+
+import java.util.List;
+import org.tensorflow.ConcreteFunction;
+import org.tensorflow.Operand;
+import org.tensorflow.op.risc.RiscAbs;
+import org.tensorflow.op.risc.RiscAdd;
+import org.tensorflow.op.risc.RiscBinaryArithmetic;
+import org.tensorflow.op.risc.RiscBinaryComparison;
+import org.tensorflow.op.risc.RiscBitcast;
+import org.tensorflow.op.risc.RiscBroadcast;
+import org.tensorflow.op.risc.RiscCast;
+import org.tensorflow.op.risc.RiscCeil;
+import org.tensorflow.op.risc.RiscCholesky;
+import org.tensorflow.op.risc.RiscConcat;
+import org.tensorflow.op.risc.RiscCondition;
+import org.tensorflow.op.risc.RiscConv;
+import org.tensorflow.op.risc.RiscCos;
+import org.tensorflow.op.risc.RiscDiv;
+import org.tensorflow.op.risc.RiscDot;
+import org.tensorflow.op.risc.RiscExp;
+import org.tensorflow.op.risc.RiscFft;
+import org.tensorflow.op.risc.RiscFloor;
+import org.tensorflow.op.risc.RiscGather;
+import org.tensorflow.op.risc.RiscImag;
+import org.tensorflow.op.risc.RiscIsFinite;
+import org.tensorflow.op.risc.RiscLog;
+import org.tensorflow.op.risc.RiscLogicalAnd;
+import org.tensorflow.op.risc.RiscLogicalNot;
+import org.tensorflow.op.risc.RiscLogicalOr;
+import org.tensorflow.op.risc.RiscMax;
+import org.tensorflow.op.risc.RiscMin;
+import org.tensorflow.op.risc.RiscMul;
+import org.tensorflow.op.risc.RiscNeg;
+import org.tensorflow.op.risc.RiscPad;
+import org.tensorflow.op.risc.RiscPool;
+import org.tensorflow.op.risc.RiscPow;
+import org.tensorflow.op.risc.RiscRandomUniform;
+import org.tensorflow.op.risc.RiscReal;
+import org.tensorflow.op.risc.RiscReduce;
+import org.tensorflow.op.risc.RiscRem;
+import org.tensorflow.op.risc.RiscReshape;
+import org.tensorflow.op.risc.RiscReverse;
+import org.tensorflow.op.risc.RiscScatter;
+import org.tensorflow.op.risc.RiscShape;
+import org.tensorflow.op.risc.RiscSign;
+import org.tensorflow.op.risc.RiscSlice;
+import org.tensorflow.op.risc.RiscSort;
+import org.tensorflow.op.risc.RiscSqueeze;
+import org.tensorflow.op.risc.RiscSub;
+import org.tensorflow.op.risc.RiscTranspose;
+import org.tensorflow.op.risc.RiscTriangularSolve;
+import org.tensorflow.op.risc.RiscUnary;
+import org.tensorflow.op.risc.RiscWhile;
+import org.tensorflow.types.TBool;
+import org.tensorflow.types.TFloat32;
+import org.tensorflow.types.TInt32;
+import org.tensorflow.types.family.TNumber;
+import org.tensorflow.types.family.TType;
+
+/**
+ * An API for building {@code risc} operations as {@link Op Op}s
+ *
+ * @see {@link Ops}
+ */
+public final class RiscOps {
```

Comment:
Any idea what this is for? It looks like it's just some core ops but for a different CPU, but we don't know what CPU.

Reply:
Google (the search engine) tells me it is related to TensorFlow Lite on RISC-V: https://hackmd.io/@nctu-cas-lab/BJ75A0IsK

So is not making much sense for TFJava?

Reply:
Yeah, I think we should hide those.